dev_guide/builds/build_strategies.adoc

```diff
@@ -525,3 +525,42 @@ spec:
     file to use, relative to the source `contextDir`.
     If `contextDir` is omitted, it defaults to the root of the repository.
     If `jenkinsfilePath` is omitted, it defaults to *_Jenkinsfile_*.
+
+[[jenkins-pipeline-strategy-environment]]
+=== Environment Variables
+
+To make environment variables available to the
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline build]
+process, you can add environment variables to the `jenkinsPipelineStrategy` definition
+of the `BuildConfig`.
+
+The environment variables defined there are set as parameters on the job in Jenkins associated with
+the `BuildConfig`. 
+
+For example:
+
+[source,yaml]
+----
+jenkinsPipelineStrategy:
+...
+  env:
+    - name: "FOO"
+      value: "BAR"
+
+----
+
+A couple of details regarding the interaction between Jenkins job parameters and `BuildConfig` environment variables:
+
+1. The value of environment variable in the `BuildConfig` becomes the default value of the parameter in the Jenkins job
+2. The environment variable values in the `BuildConfig` will override any changes to the job parameter default values you make
+from the Jenkins console when starting the job via `oc start-build`; if you start the job from the Jenkins console instead, you 
+can then control the values from the Jenkins console
```

Comment:
Yes there is a precedence.  But how you start the job also matters (`oc start-build` vs. Jenkins console).  

I'll break out this bullet into multiple bullets based on how the job is started (one for `oc start-build` and one for Jenkins console) and see if that helps.